contracts/base/LendingPool.sol

```diff
@@ -53,6 +60,8 @@ contract LendingPool is Base, LendingPoolInterface {
 
     MarketStateLib.MarketState internal compoundMarketState;
 
+    IERC20 private compToken;
```

Comment:
Lets not store it on the LendingPool but rather fetch it from the AssetsController via the mapping: `assetBySymbol('COMP')`

Reply:
Forgot to delete that